.prettierrc

```
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 120,
+  "trailingComma": "all",
+  "bracketSameLine": true,
+  "endOfLine": "auto"
```

```suggestion
  "singleQuote": true,
  "endOfLine": "auto"
```
따옴표 설정에 대한 부분이 누락되어 추가 부탁드립니다 ✨